### PR TITLE
feat: show errors and syncing indicators

### LINKED ISSUE #575: Sync indicators + Errors indicator + Retry button

### PR DESCRIPTION
**Pull request by cazala**
Closes #575 

- Syncing indicators
- Error indicators
- Retry sync button

errors-cloudsaves

### CODE REVIEW

File: src/components/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import LoadingPage from 'components/LoadingPage'
 import PromoBanner from './PromoBanner'
 import { Props, State, DefaultProps } from './HomePage.types'
 import './HomePage.css'
+import SyncToast from 'components/SyncToast'
```
Comment (belohlavek) [inline src/components/HomePage/HomePage.tsx]: import order 🌞

File: src/components/ProjectCard/ProjectCard.css [hunk 2]
```diff
@@ -97,3 +97,13 @@
   margin-left: 4px;
   margin-top: 2px;
 }
+
+.ProjectCard .error-indicator {
+  width: 16px;
+  height: 16px;
+  background: url(/static/media/save-error.30420aec.svg);
```
Comment (belohlavek) [inline src/components/ProjectCard/ProjectCard.css]: /static/ ? 🌞

File: src/components/SyncToast/SyncToast.container.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux'
+
+import { RootState } from 'modules/common/types'
+import { MapStateProps, MapDispatchProps, MapDispatch } from './SyncToast.types'
```
Comment (belohlavek) [inline src/components/SyncToast/SyncToast.container.ts]: import order 🌞

File: src/components/SyncToast/SyncToast.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.SyncToast .message .close {
+  margin-left: 10px;
+}
+
+.SyncToast .error-indicator {
+  width: 16px;
+  height: 16px;
+  background: url(/static/media/save-error.30420aec.svg);
```
Comment (belohlavek) [inline src/components/SyncToast/SyncToast.css]: static 🌞

File: src/components/SyncToast/SyncToast.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
+import * as React from 'react'
+
+import { Props, State } from './SyncToast.types'
+
+import './SyncToast.css'
```
Comment (belohlavek) [inline src/components/SyncToast/SyncToast.tsx]: Juanca me va a matar un dia de estos

File: src/components/SyncToast/SyncToast.tsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        </div>
+      )
+    } else if (errorCount > 0) {
+      return (
+        <div className="SyncToast">
+          <div className="message">
+            <div className="error-indicator" />
+            {t('sync.error', { errorCount })}
+            <Button basic onClick={this.handleRetry}>
+              Retry
```
Comment (belohlavek) [inline src/components/SyncToast/SyncToast.tsx]: translation

File: src/components/ProjectCard/ProjectCard.css (hunk 2 above)
Comment (cazala) [reply src/components/ProjectCard/ProjectCard.css]: que carajos

**Pull request merged**